src/client/src/components/Image/Image.jsx

```diff
@@ -0,0 +1,26 @@
+import React from "react";
+import PropTypes from "prop-types";
+import "./image.css";
+
+export const Image = ({ src, alt, onClick }) => {
+  return (
+    <img
+      src={src}
+      className="list-item-delete-button"
+      alt={alt}
+      onClick={onClick}
+    ></img>
+  );
+};
+
+Image.propTypes = {
+  src: PropTypes.string,
+  alt: PropTypes.string,
+  onClick: PropTypes.func,
+};
+
+Image.defaultProps = {
+  src: "",
+  alt: "",
+  onClick: undefined,
```

Comment:
pretty sure that this will happen anyways if you don't define any default - but in general undefined is never something we should set explicitly since the whole point of undefined is to identify somethign that wasn't set anywhere. If we set somethign to have no value we should use null